tools/code-generation/smithy/codegen/cpp-smoke-tests-codegen/src/main/java/com/amazonaws/util/awsclientsmithygenerator/generators/SmokeTestData.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+package com.amazonaws.util.awsclientsmithygenerator.generators;
+
+import lombok.Data;
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+import software.amazon.smithy.codegen.core.Symbol;
+@Data
+public final class SmokeTestData {
+    public String testcaseName;
+    public ClientConfiguration config;
+    public String operationName;
+    public String inputShapeName;
+    public String outputShapeName;
+    public List<String> testDataCodeBlock;
+    public String requestBlock;
+    public boolean expectSuccess;
+    public Optional<String> errorShapeId;
+    //capture auth scheme as that decides the client constructor
+    String auth;
```

Comment:
nit: why is this member the only one with a comment and the only one without access modifier?

Reply:
ack. missed the specifier, shall add . The comment for this field is because this is a derived field, not parsed directly from the smoke test trait in model since its available in the service trait